### PR TITLE
Feat/allow infuxdb2 tag override

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

**Comment by rawkode** [comment]
Hopefully this gets everything passing. There's a discrepency between the Quay image and the Hub image, annoyingly.

**Comment by rawkode** [comment]
Decided to do a major bump, as there's some risk switching the images.

**Review by timhallinflux — APPROVED** [review]

**Comment by timhallinflux** [comment]
Lots of changes were made to the final image that was posted to DockerHub. This was in order to facilitate upgrades from 1.x to 2.x. 

Worth reviewing all the gory details here: https://hub.docker.com/_/influxdb

File: charts/influxdb2/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ name: influxdb2
 description: A Helm chart for InfluxDB v2
 home: https://www.influxdata.com/products/influxdb-overview/influxdb-2-0/
 type: application
-version: 1.0.17
+version: 2.0.0
```
Comment (danxmoran) [inline charts/influxdb2/Chart.yaml:8]: Since there's no `2.0.0` on DockerHub, should this be `2.0` or `2.0.4`? That way the fallback case will work.

**Pull request closed without merging**